benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -80,46 +80,47 @@ msgstr ""
 "benefit to the rider's contactless card of choice."
 
 msgid "core.pages.help.payment_options"
-msgstr "Payment options"
+msgstr "What kind of credit card can I use?"
 
-msgid "core.pages.help.payment_options.p[0]"
+msgid "eligibility.pages.start.modal.p[0]"
 msgstr ""
-"In order to activate your discount, you need a bank card that has the "
-"contactless symbol on it. The contactless symbol is four curved lines that "
-"looks like this:"
+"Benefits only works with contactless debit or credit cards. We do not "
+"currently support mobile wallets or digital cards, such as Apple Pay, Google "
+"Wallet, or Samsung Wallet."
+
+msgid "eligibility.pages.start.modal.p[1]"
+msgstr "The contactless symbol is four curved lines, like this:"
 
 msgctxt "image alt text"
 msgid "core.icons.contactless"
 msgstr "Four curved lines on contactless-enabled cards"
 
-msgid "core.pages.help.payment_options.p[1]"
-msgstr ""
-"A contactless bank card can be a credit card or a debit card and must "
-"include a Visa or Mastercard logo."
+msgid "eligibility.pages.start.modal.p[2]"
+msgstr "Your card must include a Visa or Mastercard logo."
 
-#, python-format
-msgid "core.pages.help.payment_options.p[2][0]%(website)s"
+msgid "eligibility.pages.start.modal.p[3]"
 msgstr ""
-"Don’t have access to a bank card? You can request a contactless card from "
-"one of the companies that offer free contactless prepaid debit cards, such "
-"as the <a href=\"%(website)s\" target=\"_blank\" rel=\"noopener noreferrer"
-"\">Cash App Visa Debit Card</a> "
+"Don’t have access to a contactless card? You can request a contactless card "
+"from your bank, or one of the companies that offer free contactless prepaid "
+"debit cards, such as the Cash App Visa debit card or the Venmo Mastercard "
+"debit card."
 
 #, python-format
-msgid "core.pages.help.payment_options.p[2][1]%(website)s"
+msgid "eligibility.pages.start.modal.p[4.1]%(agency_short_name)s"
 msgstr ""
-"or the <a href=\"%(website)s\" target=\"_blank\" rel=\"noopener noreferrer"
-"\">Venmo Mastercard Debit Card</a>."
+"You can still get your transit benefit by going through your "
+"%(agency_short_name)s application process. For updates on additional "
+"options, please check back on this website, or contact "
 
-msgid "core.pages.help.payment_options.p[3]"
+#, python-format
+msgid "eligibility.pages.start.modal.p[4.2]%(agency_info_url)s"
 msgstr ""
-"You can still get your discount by going through your local transit "
-"provider’s application process."
+"<a href=\"%(agency_info_url)s\" target=\"_blank\" rel=\"noopener noreferrer"
```

Comment:
Maybe the link does need to be here, because the word `contact` is supposed to be part of the link text, and that will be translated.

But it looks like the word `contact` is part of the previous message and not this one.